resources/views/Widgets/OrderConfirmation/OrderDocumentsWidget.twig

```diff
@@ -0,0 +1,80 @@
+{% import "Ceres::Widgets.Helper.TwigBuilder" as Twig %}
+{% import "Ceres::Widgets.Helper.WidgetHelper" as WidgetHelper %}
+
+{% set customClass = widget.settings.customClass.mobile %}
+{% set spacingSettings = widget.settings.spacing %}
+{% set inlinePadding   = WidgetHelper.getInlineSpacings(spacingSettings, spacingSettings.customPadding.mobile, "") %}
+{% set inlineMargin    = WidgetHelper.getInlineSpacings(spacingSettings, "", spacingSettings.customMargin.mobile) %}
+{% set spacingPadding  = WidgetHelper.getSpacingClasses(spacingSettings, spacingSettings.customPadding.mobile, "") %}
+{% set spacingMargin   = WidgetHelper.getSpacingClasses(spacingSettings, "",spacingSettings.customMargin.mobile) %}
+
+{% set appearance = widget.settings.appearance.mobile | default("primary") %}
+{% set buttonSize = widget.settings.buttonSize.mobile | default("") %}
+
+{{ Twig.set("enabledRoutes", Twig.call("config", ["IO.routing.enabled_routes"]) ~ " | split(\", \")") }}
+{{ Twig.set("translationMap",
+    {
+        invoice:            "Ceres::Template.orderConfirmationOrderDocumentsInvoice",
+        invoice_external:   "Ceres::Template.orderConfirmationOrderDocumentsInvoiceExternal",
+        delivery_note:      "Ceres::Template.orderConfirmationOrderDocumentsDeliveryNote",
+        order_confirmation: "Ceres::Template.orderConfirmationOrderDocumentsOrderConfirmation",
+        pickup_delivery:    "Ceres::Template.orderConfirmationOrderDocumentsPickupDelivery",
+        reversal_document:  "Ceres::Template.orderConfirmationOrderDocumentsReversalDocument"
+    } | json_encode)
+}}
+
+{{ Twig.set("orderDocuments", "[]") }}
+{{ Twig.for("document", "data.order.documents | default([])", "translationMap[document.type] is defined") }}
+    {{ Twig.set("orderDocuments", "orderDocuments | merge([document])") }}
+{{ Twig.endfor() }}
+
+{{ Twig.if("('order-document' in enabledRoutes or 'all' in enabledRoutes) and (orderDocuments | length > 0 or #{ isPreview | json_encode })") }}
+    <div class="widget widget-order-documents widget-{{ appearance }}{% if customClass | length > 0 %} {{ customClass }}{% endif %}{% if spacingMargin | length > 0 %} {{ spacingMargin }}{% endif %}"
+            {% if inlineMargin | length > 0 %} style="{{ inlineMargin }}"{% endif %}>
+
+        {{ Twig.if("orderDocuments | length > 1 and ('order-document' in enabledRoutes or 'all' in enabledRoutes) or #{ isPreview | json_encode }") }}
```

Comment:
`('order-document' in enabledRoutes or 'all' in enabledRoutes) or #{ isPreview | json_encode }` wird bereits in Zeile 31 abgefragt und ist daher an dieser Stelle immer `true`

Reply:
Den Part mit der Route habe ich entfernt, dieser war unnötig. isPreview ist hier, damit man in der Vorschau trotzdem den Button sehen kann, auch wenn es keine Dokumente gibt.

Hier findest du die Änderungen: https://github.com/plentymarkets/plugin-ceres/pull/1361/commits/0b85a51afa50a6fc7fb743e6e7a0cfe4b07be62c